command/src/state.rs

```diff
@@ -379,22 +374,24 @@ impl ConfigState {
             .parse()
             .with_context(|| "Could not parse socket address")?;
 
+        let old_fingerprint = Fingerprint(
+            hex::decode(&replace.old_fingerprint)
+                .with_context(|| "Failed at decoding the string (expected hexadecimal data)")?,
```

Comment:
We have to return the error

Reply:
I don't get it. This code *does* return the error.